pkg/kubelet/dockershim/network/plugins.go

```diff
@@ -63,7 +64,7 @@ type NetworkPlugin interface {
 	// SetUpPod is the method called after the infra container of
 	// the pod has been created but before the other containers of the
 	// pod are launched.
-	SetUpPod(namespace string, name string, podSandboxID kubecontainer.ContainerID, annotations map[string]string) error
+	SetUpPod(namespace string, name string, podSandboxID kubecontainer.ContainerID, annotations map[string]string, dnsConfig *runtimeapi.DNSConfig) error
```

Comment:
Using the type from CRI as input seems a bit weird. But network plugin these days are already dedicated to dockershim. 

This just make it even harder to be portable.

Reply:
@freehan Thanks for reviewing. Let me replace it with an option with map type instead (in case we would add other options in the future).